pyproject.toml

```diff
@@ -0,0 +1,25 @@
+#===============================================================================
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+
+[tool.black]
+line-length = 90
+target-version = ['py38', 'py39', 'py310', 'py311', 'py312']
```

Comment:
Do we have CI runs for these versions? Azure Pipelines currently looks hardcoded to 3.10, but I remember having issues running 3.8 or 3.9 last summer. Some of the testing may conflict with versions without dpnp/dpctl support (i.e. 3.8, 3.11 3.12). I'd like to see CI jobs with  those versions green.

Reply:
CI matrix is updated to use 3.9 - 3.11 version. `modin-all` packages are not available for 3.12 yet.